src/kernel/heap.zig

```diff
@@ -100,8 +99,12 @@ pub const FreeListAllocator = struct {
     /// Arguments:
     ///     IN self: *FreeListAllocator - The allocator being freed within
     ///     IN mem: []u8 - The memory to free
+    ///     IN alignment: u29 - The alignment used to allocate the memory
+    ///     IN ret_addr: usize - The return address passed by the high-level Allocator API. This is ignored.
     ///
-    fn free(self: *FreeListAllocator, mem: []u8) void {
+    fn free(self: *FreeListAllocator, mem: []u8, alignment: u29, ret_addr: usize) void {
```

Comment:
Why not use the `Self` defined?

Reply:
That is actually within the `Header` struct, but I can add one for `FreeListAllocator` too.